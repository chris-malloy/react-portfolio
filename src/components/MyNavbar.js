import React from 'react';
import { Navbar, NavItem } from 'react-materialize';

const MyNavbar = () => {
  return (
    <Navbar brand='Chris Malloy' className="blue" right>
      <NavItem href='/#projects'>Projects</NavItem>
      <NavItem href='/#skills'>Skills</NavItem>
      <NavItem href='/#my-story'>My Story</NavItem>
      <NavItem href='/#contact'>Contact</NavItem>
      <NavItem href='images/chris-malloy-resume-main.pdf'>Resume</NavItem>
    </Navbar>
  );
};

export default MyNavbar;