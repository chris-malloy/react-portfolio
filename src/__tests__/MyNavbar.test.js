import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import MyNavbar from '../components/MyNavbar';

configure({ adapter: new Adapter() });

describe('Navbar', () => {
  const navbar = shallow(<MyNavbar />);

  it('renders correctly', () => {
    expect(navbar).toMatchSnapshot();
  });

  
});