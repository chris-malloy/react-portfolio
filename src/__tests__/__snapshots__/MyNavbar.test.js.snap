// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyNavbar renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MyNavbar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "brand": "Chris Malloy",
      "children": Array [
        <NavItem
          href="/#projects"
>
          Projects
</NavItem>,
        <NavItem
          href="/#skills"
>
          Skills
</NavItem>,
        <NavItem
          href="/#my-story"
>
          My Story
</NavItem>,
        <NavItem
          href="/#contact"
>
          Contact
</NavItem>,
        <NavItem
          href="images/chris-malloy-resume-main.pdf"
>
          Resume
</NavItem>,
      ],
      "className": "blue",
      "href": "/",
      "options": Object {},
      "right": true,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "Projects",
          "href": "/#projects",
        },
        "ref": null,
        "rendered": "Projects",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "Skills",
          "href": "/#skills",
        },
        "ref": null,
        "rendered": "Skills",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "My Story",
          "href": "/#my-story",
        },
        "ref": null,
        "rendered": "My Story",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "Contact",
          "href": "/#contact",
        },
        "ref": null,
        "rendered": "Contact",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "Resume",
          "href": "images/chris-malloy-resume-main.pdf",
        },
        "ref": null,
        "rendered": "Resume",
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "brand": "Chris Malloy",
        "children": Array [
          <NavItem
            href="/#projects"
>
            Projects
</NavItem>,
          <NavItem
            href="/#skills"
>
            Skills
</NavItem>,
          <NavItem
            href="/#my-story"
>
            My Story
</NavItem>,
          <NavItem
            href="/#contact"
>
            Contact
</NavItem>,
          <NavItem
            href="images/chris-malloy-resume-main.pdf"
>
            Resume
</NavItem>,
        ],
        "className": "blue",
        "href": "/",
        "options": Object {},
        "right": true,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "Projects",
            "href": "/#projects",
          },
          "ref": null,
          "rendered": "Projects",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "Skills",
            "href": "/#skills",
          },
          "ref": null,
          "rendered": "Skills",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "My Story",
            "href": "/#my-story",
          },
          "ref": null,
          "rendered": "My Story",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "Contact",
            "href": "/#contact",
          },
          "ref": null,
          "rendered": "Contact",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "Resume",
            "href": "images/chris-malloy-resume-main.pdf",
          },
          "ref": null,
          "rendered": "Resume",
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
